spark/dl/src/main/scala/com/intel/analytics/bigdl/tensor/DenseTensor.scala

```diff
@@ -1717,24 +1720,62 @@ private[tensor] class DenseTensor[@specialized T: ClassTag](
     val indicesTensor = if (indices == null) Tensor[T]() else indices
     indicesTensor.resize(topKSize)
 
+    @inline
+    def compare(a: T, b: T): Boolean = ev.isGreater(b, a)  ^ !increase
+    /* if (increase) {
```

Comment:
Why comment it instead of deleting it ?

Reply:
Much easier to read it.

Reply:
We need to remove it as commented code, probably you may want to add comment on top of the function